app/src/main/java/io/intrepid/contest/screens/contestjudging/scoreentrydetail/RateEntryFragment.java

```diff
@@ -0,0 +1,83 @@
+package io.intrepid.contest.screens.contestjudging.scoreentrydetail;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.widget.ImageView;
+
+import com.squareup.picasso.Picasso;
+
+import java.util.List;
+
+import butterknife.BindView;
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BaseFragment;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Category;
+import io.intrepid.contest.models.Entry;
+import io.intrepid.contest.screens.contestjudging.scoreentrydetail.expandablerecycler.ScorableCategoryAdapter;
+
+import static io.intrepid.contest.screens.contestjudging.scoreentrydetail.ScoreEntryDetailActivity.ENTRY_EXTRA_KEY;
+
+public class RateEntryFragment extends BaseFragment<RateEntryContract.Presenter> implements RateEntryContract.View {
+    @BindView(R.id.top_horizontal_entry_image_card)
+    ImageView topImageCard;
+    @BindView(R.id.generic_recycler_view)
+    RecyclerView categoriesRecyclerView;
+    private ScorableCategoryAdapter scorableCategoryAdapter;
+
+    public static Fragment newInstance(Entry entry) {
```

Comment:
This is a valid point, and it's definitely doable. But even before that, how would we handle going from the list of entries in SubmissionEntriesActivity to ScoreEntryDetailActivity? At some point, we will have to go from one activity to another with and pass along that single entry to be displayed.